x-pack/plugins/timelines/public/components/t_grid/standalone/index.tsx

```diff
@@ -341,7 +362,25 @@ const TGridStandaloneComponent: React.FC<TGridStandaloneProps> = ({
       isFirstUpdate.current = false;
     }
   }, [loading]);
-  const timelineContext = { timelineId: STANDALONE_ID };
+
+  useEffect(() => {
```

Comment:
couldn't this logic just happen in the two places setSelectedAlertIndex is called?